sections/sprint-3/dead-reckoning.tex

```diff
@@ -0,0 +1,84 @@
+\section{Dead reckoning}
+The accuracy of the representation of the players as well as the ball is important for the game to function well.
+Since these positions are sent over the network, they might not always be the most recent positions, since they could be delay or packages lost.
```

Comment:
```suggestion
Since these positions are sent over the network, they might not always be the most recent positions, since there could be delays or packages could be lost.
```